change-version.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -eux
+
+# specify new version as version number, optionally followed by
+# '-SNAPSHOT' for a snapshot version
+
+if [ "$#" != 2 ]; then
+  echo "Usage: $0 <old-version> <new-version>"
+  exit 1
+fi
+
+replace() {
+  pattern=$1
+  replacement=$2
+  shift 2
+
+  find . \( "$@" \) -print0 |
+    xargs --null --max-args=1 sed --in-place --expression="s/$pattern/$replacement/g"
```

Comment:
So I used `perl` before because `sed` on Mac and Linux have some subtle differences (GNU vs. BSD I think).  It would be good to test this command on both platforms.  Let me know if you need help testing on Mac

Reply:
Thanks for bringing this up, @msridhar. Yes, certain GNUisms will need to be changed. I will push a revised variant when ready, at which time I will greatly appreciate some macOS testing help from you. (Not yet, thought!)

By the way, the easy part of the changes will be to replace `--long-options` with single-character flags. The harder part is using `sed` for in-place rewriting, since the Linux and macOS flags for this are mutually incompatible. For that I will likely revert to using `perl` as you were doing before.